decidim-blogs/app/controllers/decidim/blogs/posts_controller.rb

```diff
@@ -91,17 +91,17 @@ def post_presenter
 
       def posts
         @posts ||= if current_user&.admin?
-                     Post.where(component: current_component)
+                     Post.where(component: current_component).published_at_desc
                    else
-                     Post.published.where(component: current_component)
+                     Post.published.where(component: current_component).published_at_desc
                    end
       end
 
       # PROVISIONAL if we implement counter cache
       def posts_most_commented
         @posts_most_commented ||= posts.joins(:comments).group(:id)
                                        .select("count(decidim_comments_comments.id) as counter")
-                                       .select("decidim_blogs_posts.*").order("counter DESC").created_at_desc.limit(7)
+                                       .select("decidim_blogs_posts.*").order("counter DESC").published_at_desc.limit(7)
```

Comment:
if we order by most recent, there is no point on keeping ordering by counter. 

```suggestion
                                       .select("decidim_blogs_posts.*").published_at_desc.limit(7)
```

Reply:
Actually this part only applies to a filter to order based on the number of comments. I think it should stay as it is!

Reply:
Oh, i see it now.